docs/01-getting-started.md

```diff
@@ -2,21 +2,23 @@
 
 ## Creating Your First Project
 
-The quickest way to get up and running with Studio is to clone an existing Git repository. If you're running from the Studio desktop version, then this can be as easy as choosing a location on your computer where a repository already exists. If you're running from the browser, this can be as simple as pasting the URL to any Github repository that you'd like to add. Give it a shot with a sample project we've provided below.
+The quickest way to get up and running with Studio is to clone an existing Git repository. If you're running Studio Desktop then this could be choosing a location on your computer where a repository already exists. If you're running Studio Web you could copy the URL to any GitHub, Bitbucket, GitLab, Gitea, or other Git repository. 
```

Comment:
Studio Web doesn't offer clone via URL, but the Desktop does. Studio Web offers adding projects via VCS oauth.

```suggestion
The quickest way to get up and running with Studio is to clone an existing Git repository. If you're running Studio Desktop then this could be choosing a location on your computer where a repository already exists, or you could copy the URL to any GitHub, Bitbucket, GitLab, Gitea, or other Git repository.
```